src/api/NetworkErrorRetryLink.js

```diff
@@ -0,0 +1,5 @@
+import { RetryLink } from '@apollo/client/link/retry';
+
+export default () => {
+	return new RetryLink();
```

Comment:
Do we need to add any defaults (and configurable via env vars so changes are easier)?

Reply:
I don't believe so. They document their defults in the link above.

Reply:
There's a max of 5 and also delay mechanisms...

Reply:
OK I only quickly read it and didn't catch that those were defaults.

Reply:
With max of 5 I'm wondering if that's too much, because it could tie up workers long enough for pending tasks to pile up. Maybe we should reduce it to 1 or two or make it configurable?

Reply:
Looking at adding a couple configs to allow us to control the attempts and activate/deactivate retries